Tetris/src/tetris/GameRunner.java

```diff
@@ -30,20 +47,90 @@ static void startGame(){
 	 * @param user input
 	 */
 	static void inputListener(){
-		// move
+		// Listener: Press A
+			// Behavior: Rotate counterclockwise
+		// Listener: Press D
+			// Behavior: Rotate clockwise
 		
-		// rotate
+		// Listener: Press Left arrow
+			// Behavior: Shift Tetromino left 
 		
-		// drop
+		// Listener: Press Right arrow
+			// Behavior: Shift Tetromino right
 		
-		// pause
```

Comment:
Something is off here in the description. Rotate is A and D, shift should be left arrow and right arrow, drop should be down arrow, pause = spacebar, and we nixed using "move" because it is vague...I think some of these labels might just be leftovers.